src/clients/tfgmAPI.ts

```diff
@@ -0,0 +1,89 @@
+import { config } from '../config'
+
+const tfgmEndpoint = 'https://api.tfgm.com/odata/Metrolinks'
+
+export type TfGMData = {
+  AtcoCode: string
+  StationLocation: string
+  Dest0: string
+  Wait0: string
+  Status0: string
+  Carriages0: string
+  Dest1: string
+  Wait1: string
+  Status1: string
+  Carriages1: string
+  Dest2: string
+  Wait2: string
+  Status2: string
+  Carriages2: string
+}[]
+
+class StopData {
+  stopName = ''
+  incomingTrams: IncomingTram[] = []
+}
```

Comment:
There's not much benefit to using classes and I think we generally just define types and have objects of that type